docs/connect.html

```diff
@@ -53,7 +53,7 @@ <h4><a id="connect_running" href="#connect_running">Running Kafka Connect</a></h
 
     <p>The important configuration options specific to standalone mode are:</p>
     <ul>
-        <li><code>offset.storage.file.filename</code> - File to store offset data in</li>
+        <li><code>offset.storage.file.filename</code> - File to store source task offsets</li>
```

Comment:
Same suggestion RE "connector" vs. "task":
```suggestion
        <li><code>offset.storage.file.filename</code> - File to store source connector offsets</li>
```

Reply:
Thanks.